### DISCUSSION TITLE
Truenas Scale Truecharts Custom-App

### DISCUSSION THREAD

**Discussion by TimothyWright95** [RandomNinjaAtk/arr-scripts#71] [Q&A]
Has anyone been able to get this to work in a custom true-charts app on truenas scale. Ive been trying for a while and keep having issues with permissions as well as non existent files. Everything apears to be working fine until about line 640ish then a bunch of No such File or Directory errors pop up and im not sure if thats normal or an error. Either way it doesnt seem to work. 
[custom-app_custom-app-654557c7f4-9mnv6_custom-app.log](https://github.com/RandomNinjaAtk/arr-scripts/files/12482256/custom-app_custom-app-654557c7f4-9mnv6_custom-app.log)

I have the /custom-services.d and /custom-cont-init.d shares mounted via NFS share which i think might be the issue, but unfortunately thats the only way for truecharts apps to mount part of the file system into the container. 

Instalation instructions seem fairly straight forward but perhaps im missing something?

**Comment by RandomNinjaAtk** [comment]
I don't have any experience with truenas to give you any kind of help. However, I know that truenas can sometimes cause issues with the Arr apps, just based on what I've observed in community chatter...
